src/gui/qgsgeometryrubberband.h

```diff
@@ -86,12 +92,12 @@ class GUI_EXPORT QgsGeometryRubberBand: public QgsMapCanvasItem
     };
 
     QgsGeometryRubberBand( QgsMapCanvas *mapCanvas, QgsWkbTypes::GeometryType geomType = QgsWkbTypes::LineGeometry );
-    ~QgsGeometryRubberBand() override;
+    virtual ~QgsGeometryRubberBand() override;
```

Comment:
This shouldn't be needed -- it's already virtual

Reply:
Indeed! I believed I already removed this...